config/tailwind.config.js

```diff
@@ -11,8 +11,23 @@ module.exports = {
   theme: {
     extend: {
       fontFamily: {
-        sans: ['Inter var', ...defaultTheme.fontFamily.sans],
+        sans: ['Poppins', ...defaultTheme.fontFamily.sans],
       },
+      colors: {
+        'primary-300': '#E8E6F6',
+        'primary-500': '#8378DF',
+        'primary-600': '#726DD9',
+        'primary-700': '#5F5FC9',
+        'light-gray': '#F9FAFC',
+        'gray-2': '#1F2D3D',
```

Comment:
You should be able to further simplify the color schema by defining the colors and shades as an object like so:
```
'primary': {
  300: '#E8E6F6',
  400: '#E8E6F6',
  ...
},
'gray': {
  '2': '#1F2D3D',
  '3':  '#3C48BF',
  ...
}
```